.github/workflows/build_container.yaml

```diff
@@ -1,85 +1,84 @@
           auto-update-conda: true
           channels: conda-forge,defaults
           miniforge-variant: Mambaforge
           use-mamba: true
           channel-priority: strict
           show-channel-urls: true
 
-      - name: configure conda and install requirements
+      - name: Configure conda and install requirements
         id: rs-install
         shell: bash -l {0}
         run: |
           mamba install --quiet --file=requirements.txt
           mamba install --quiet --file=test-requirements.txt
-          mamba list rubin-sim | grep -v "#" | awk '{print $2}' > ${{ github.workspace }}/rs_version
+          mamba list rubin-scheduler | grep -v "#" | awk '{print $2}' > ${{ github.workspace }}/rs_version
           echo "rs-version" `cat ${{ github.workspace }}/rs_version`
-          echo "rs-version=`cat ${{ github.workspace }}/rs_version`" >> $GITHUB_OUTPUT
+          echo "rs-version=`cat ${{ github.workspace }}/rs_version`" >> $GITHUB_OUTPUT   
 
-      - name: Access rubin-sim-data cache
+      - name: Access rubin-sched-data cache
         id: cache-rs
         uses: actions/cache@v3
         env:
-          cache-name: cached-rubin-sim-data
+          cache-name: cached-rubin-sched-data
         with:
-          path: ${{ github.workspace }}/rubin_sim_data
+          path: ~/rubin_sim_data
           key: ${{ env.cache-name }}-v${{ steps.rs-install.outputs.rs-version }}
           restore-keys: |
             ${{ env.cache-name }}-v
 
-      - if: ${{ steps.cache-rs.outputs.cache-hit == 'true' }}
-        name: List cache contents.
-        shell: bash -l {0}
-        run: |
-          export RUBIN_SIM_DATA_DIR=${{ github.workspace }}/rubin_sim_data
-          echo $RUBIN_SIM_DATA_DIR
-          ls $RUBIN_SIM_DATA_DIR
-          echo "__contents of versions.txt__"
-          cat $RUBIN_SIM_DATA_DIR/versions.txt
-
-      - if: ${{ steps.cache-rs.outputs.cache-hit != 'true' }}
-        name: Download data.
+      - name: Check and update cache contents.
         shell: bash -l {0}
         run: |
-          export RUBIN_SIM_DATA_DIR=${{ github.workspace }}/rubin_sim_data
-          rs_download_data --force --dirs scheduler,site_models,skybrightness_pre,throughputs,tests --tdqm_disable
+          export RUBIN_SIM_DATA_DIR=~/rubin_sim_data
+          scheduler_download_data --update --tdqm_disable
 
-      - name: report environment
+      - name: Report environment
         shell: bash -l {0}
         run: |
           echo pwd is `pwd`
           echo python is `which python`
           echo github.workspace is ${{ github.workspace }}
           find ${{ github.workspace }} -maxdepth 2
 
-      - uses: lsst-sqre/build-and-push-to-ghcr@v1
+      - uses: lsst-sqre/build-and-push-to-ghcr@tickets/DM-41857
         id: build
         with:
           image: ${{ github.repository }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
           dockerfile: Dockerfile
+          cache-from: type=local,src=/tmp/.buildx-cache
```

Comment:
So .. the defaults (from v1, without using cache-from, cache-to) puts the build-kit-blah-blah blah into *our* GitHub Actions cache, one per each run basically (because it's keyed to the entire sha). 
This quickly filled our cache. 
This would not necessarily be a problem (which is why sqr left it as the default before) because old caches are just evicted, so "whatever". 
However, since we actually use our cache for things, it could mean that our desired cache contents get evicted when running and testing on a branch and making lots of docker builds. See https://lsstc.slack.com/archives/C2JP8GGVC/p1700720935137609

The cache contents are just the docker layers, so by using "local" and /tmp, etc. we don't cache any of the docker build layers. However, I'm not entirely sure if they could use them anyway, since they were keyed to the sha (depends on details in the sqr reusable workflow and the docker workflow they built on). So anyway, we're not building anything tooo complicated I think, and if we see problems, we can figure out something else, but this seemed better for now.